### DISCUSSION TITLE
What steps/settings can I do to get suspend of Windows 11 working?

### DISCUSSION THREAD

**Discussion by drmuey** [utmapp/UTM#5848] [UTM Help (macOS)] [answered]
**Describe the issue** 

[The controls document](https://docs.getutm.app/basics/controls/) says regarding _Suspend the virtual machine when supported._:

> The suspend feature will not work if any device that is in use does not support suspend. This typically includes emulated NVMe devices as well as GPU accelerated display cards. Additionally, QEMU hypervisor on Intel does not support suspend.

Windows 11 from Crystal Fetch out of the box (i.e. w/ default options) suffers from that.

That means any time I need to use Windows I have to boot up and start the program in question (takes a ton of time), then when I am done shut down windows (also a ton of time) then quit UTM. It’d be super dope if I could get suspend working so I can simply hit play and then pause and quit UTM.

I am hoping here to find out what I can do to ensure my VM does not use any device that does not support suspend. FWiW I haven’t used Windows for like 20 years `¯\_(ツ)_/¯` so pardon any ignorance.

**Configuration**
* UTM Version: 4.4.4 (92)
* macOS Version: 13.4.1 (c) (22F770820d)
* Mac Chip (Intel, M1, ...): 2.8 GHz Quad-Core Intel Core i7
* ISO ➜  Windows 11 from Crystal Fetch (22621.1702.230505-1222.ni_release_svc_refresh_CLIENTCONSUMER_RET_x64FRE_en-us.iso to be exact)
* Spice: spice-guest-tools-0.164.4.iso
* Options: defaults + I did try changing its display from `virtio-vga-gl (GPU Supported)` to `virtio-vga` to no avail.

**Crash log**

N/A

**Debug log**

[utm-debug.log.zip](https://github.com/utmapp/UTM/files/13208336/utm-debug.log.zip)

**Upload VM**  

[config.plist.zip](https://github.com/utmapp/UTM/files/13208142/config.plist.zip)

**Comment by osy** [comment, marked as answer]
Unfortunately right now you have to install Windows to a NVMe drive. You can do it with virtio block but you need drivers and that's a bit advanced. I think in the future QEMU will have migration working for NVMe so wait for that.

**Reply by drmuey** [reply]
thank you for clarifying 👍

**Reply by scotthue** [reply]
Following up a bit over a year later.  Is there any progress on this, or an explainer on how to setup Windows on an NVMe drive? The ability to suspend is a killer feature.

**Reply by cvn** [reply]
@scotthue I got suspend working for Win11 with these instructions https://github.com/utmapp/UTM/issues/3819#issuecomment-2408811546
